src/main/scala/me/rexim/morganey/module/Module.scala

```diff
@@ -8,16 +8,26 @@ import scala.util._
 
 import java.net.{URL, URLClassLoader}
 
-class Module(modulePath: ModulePath, classLoader: ClassLoader = Module.getClass.getClassLoader) {
+class Module(
+  modulePath: ModulePath,
+  preludeModule: Option[Module] = None,
```

Comment:
This is the first thing that asked myself when I introduced this argument. Unfortunately, it is not obvious which argument will be used more often in the future. I suggest to merge as is and later it will be more clear.